src/pages/HistoryPage/style.scss

```diff
@@ -22,3 +22,13 @@
     @include rem(font-size, 30px);
   }
 }
+
+.sticky-title {
```

Comment:
`sticky-title` could be moved to the separate file (defined twice and could be reused if new pages will appear).